components/job/job-logger.vue

```diff
@@ -238,7 +238,7 @@
   };
 </script>
 
-<style lang="scss">
+<style lang="scss" scoped>
```

Comment:
The issue with the badges seems to be because this component got scoped, but also has a rule for `.badge`.
Maybe we should add a component for the badge to fix that?

Reply:
@simon-debruijn I fixed it myself, was a small change. See the commits below.
I didn't create an RFC for the component because it was pretty simple.